packages/dds/tree/src/changeset/delta.ts

```diff
@@ -0,0 +1,281 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * This format describes changes that must be applied to a document tree in order to update it. Instances of this format
```

Comment:
personally, I find large comments like this to be nicer if you line break them semantically where possible (ex: at sentence boundaries) instead of arbitrarily where they get too long. Tends to make diffs and code review tools work better (better follow the semantics).

Reply:
Good point. I have added some line breaks.